mycroft/configuration/mycroft.conf

```diff
@@ -149,13 +149,18 @@
   // Override: REMOTE
   "listener": {
     "sample_rate": 16000,
+    // Set 'save_path' to configure the location of files stored if
+    // 'record_wake_words' and/or 'save_utterances' are set to 'true'.
+    // WARNING: Make sure that user 'mycroft' has write-access on the
+    // directory!
+    "save_path": "/tmp",
```

Comment:
I wonder if this should be commented out as /tmp isn't always the temp directory. Commenting it out, it would use `gettempdir()` to determine the location which is safer.

Reply:
No problem, just pushed an updated version!